### ISSUE TITLE
Runtime errors during Director testing with Homebrew Python 2.7.13_1

### ISSUE THREAD

**Issue by m-chaturvedi** [RobotLocomotion/drake#6608]
We see `Fatal Python error: PyThreadState_Get: no current thread` errors when some Python tests are [run in director](https://drake-jenkins.csail.mit.edu/view/Continuous%20Production/job/mac-clang-continuous-debug/2458/consoleText)

And @jamiesnape pointed out that they are perhaps because of this [change in Homebrew](https://github.com/Homebrew/homebrew-core/commit/e7f4e1c03abbe0842b77e3d163e737a81ca955e7#diff-313a6a3d25aa041ee3dc3cbd65d4f22b)

**Comment by jamiesnape** [comment]
Note also that adding `/usr/local/opt/python/libexec/bin` does not fix the problem.

**Comment by jamiesnape** [comment]
FYI @stonier. As mentioned briefly in the meeting earlier. Also affects Spartan.

**Comment by patmarion** [comment]
I looked through the build log but couldn't find anything obvious, it does look like everything that is compiled is using the correct version of Python.  Is it possible that the VTK bottles are out of date or using the wrong version of Python?

I just started a Travis-CI test of director/master, since it hasn't run recently.  It should do a `brew update` so maybe the issue will show up there.

Is it possible to get SSH access to a machine where this issue is occurring?  Without interactive debugging, I think running `otool -L *.dylib` at the end of the build would be helpful information to see in the log.
